packages/web/src/components/pages/TestSuites/TestSuiteDetails/TestSuiteSettings/SettingsTests/Modals/TestModal.tsx

```diff
@@ -84,7 +86,7 @@ const TestModal: React.FC<TestModalProps> = props => {
         >
           <Select onChange={setTestValue} value={testValue} showSearch placeholder="Select a test..." size="middle">
             {allTestsData.map(item => (
-              <Option value={item.name} key={item.name} title={item.name}>
+              <Option value={item.name} key={item.name}>
```

Comment:
This `title` here is used for recognition in E2E tests:
```suggestion
              <Option value={item.name} key={item.name} title={item.name}>
```